store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/impl/AggregatorIterator.java

```diff
@@ -57,6 +57,9 @@
     public Value reduce(final Key key, final Iterator<Value> iter) {
```

Comment:
Should the `AggregatorIterator` break when reducing an empty iterator? I fixed this as I thought it was strange but perhaps it is better leaving it?

Reply:
*shrug*
if it needed/should/best to break then a comment should notify so.